Sources/StreamChatSwiftUI/ChatChannelList/SearchResultsView.swift

```diff
@@ -53,6 +53,7 @@ struct SearchResultsView<Factory: ViewFactory>: View {
                     }
                 }
             }
+            .background(Color(colors.background))
```

Comment:
Could the background also only be set on the entire `VStack` here instead of both the `HStack` and the `ScrollView`?